flipperdevices/flipperzero-firmware
# NFC Unit tests part 1.1

# What's new

- NFC-A 7byte UID file save/load test, MFC 1/4k 4/7byte UID save/load test with default values.
- nfc_device_save[_shadow] now takes just the device and the path as an argument. **Requires thorough testing.**

# Verification 

- Run `unit_tests nfc` in the CLI. All tests up to MFC 4k7b should pass on the current firmware.
- Check all the places where nfc_device_save is used, including saving files that are already existing. 

# Checklist (For Reviewer)
